static/js/notifications.js

```diff
@@ -164,24 +163,23 @@ export function is_window_focused() {
 }
 
 export function notify_above_composebox(
-    note,
-    link_class,
+    banner_text,
+    classname,
+    link_text,
     above_composebox_narrow_url,
     link_msg_id,
-    link_text,
 ) {
     const $notification = $(
         render_compose_notification({
-            note,
-            link_class,
+            banner_text,
+            classname,
+            link_text,
```

Comment:
this renames note to banner_text, and link_class to classname

Reply:
Renaming parameters in a template seems like something that should go in a prep commit.